examples/react-simple/src/ix/Script.js

```diff
@@ -1,28 +1,64 @@
 import React, {useState} from "react"
 import * as sdk from "@onflow/sdk"
+import * as t from "@onflow/types"
 
 export const Script = () => {
   const [result, setResult] = useState(null)
 
   const run = async () => {
+
+    /*
+
+      Script
+      ------
+      
+      Scripts are declared by building an interaction and specifying a script within it.
+      This is done by specifying Cadence code within the script builder.
+
+      Declaring a script that returns an integer may look like such:
+
+        sdk.script`
+          pub fun main(): Int {
+            return 1
+          }
+        `
+
+      Script Arguments
+      ----------------
+
+      Scripts can consume arguments. These arguments are populated into the main function.
+      Declaring arguments for a Script requires calling the sdk.args and sdk.arg builders like such:
+          
+        sdk.args([sdk.arg(1, t.Int), sdk.arg(2, t.Int)]),
+
+      The order of the arguments here matters, since the order will correspond to the order of the 
+      arguments in the main function of your Cadence script.
```

Comment:
```suggestion
      The order of the arguments here matters, since the order will correspond to the order which you declared your parameters in the main function of your Cadence script.
```